FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -1690,6 +1693,7 @@
 		<rdf:type rdf:resource="&lcc-cr;GeographicRegionIdentifier"/>
 		<rdfs:label>US-AA</rdfs:label>
 		<skos:definition>subdivision code for the state designation for Armed Forces Americas, excluding Canada</skos:definition>
+		<fibo-fnd-plc-adr:hasLocaleDesignation>AA</fibo-fnd-plc-adr:hasLocaleDesignation>
```

Comment:
It's unsatisfactory having a designation for the Code as opposed to the Subdivision. It could get very hard to manage and use chains of such things before you get to the real thing. 
Why not have it as another code/name for the Subregion?